waiter/src/waiter/settings.clj

```diff
@@ -86,7 +86,8 @@
                                               s/Keyword schema/require-symbol-factory-fn}
                                              schema/contains-kind-sub-map?)
    ;; TODO port belongs in server-options?
-   (s/required-key :port) schema/positive-int
+   (s/optional-key :port) schema/positive-int
+   (s/required-key :ports) [schema/positive-int]
```

Comment:
Should we move this to `server-options`?
Else, should we treat port as `hostname` where it can be a scalar or vector?

Reply:
And we should do the same for Jet.  Dynamic typing FTW

Reply:
Keep `port` or rename to `ports`?

Reply:
Keep `port` for backwards compatibility?

Reply:
Done